Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/math-functions/least.adoc

= LEAST
:description: The LEAST() function returns the least or smallest value in a list of values.
:page-topic-type: reference

The `LEAST()` function returns the least or smallest value in a list of values. It needs at least one argument to work with, and if you mix different types, like a text and a number, it returns an error.

For example, comparing the greatest value among 4, "`two`", and 9 would result in an error.

== Syntax

The syntax for the `LEAST()` function is as follows:

[source,sql]
----
LEAST(value_1, [value_n])
----

Where:

* `value_1`: Represents the first value.
* `value_n`: Represents one or more additional values, separated by commas.

[NOTE]
====
*Info:*
 -`NULL` values in the list will be ignored.
 - The result will be `NULL` if all the expressions evaluate to `NULL`.
====

== Examples

> The following are several examples of the `LEAST()` function:

Reviewer: ```suggestion
These examples show how to use the `LEAST()` function:
```

=== Basic usage

> Consider the following example:

Reviewer: ```suggestion
Consider the example:
```
